public/app/features/variables/datasource/reducer.ts

```diff
@@ -45,15 +45,19 @@ export const dataSourceVariableSlice = createSlice({
           continue;
         }
 
+        if (source.isDefault) {
+          if (regex === undefined) {
+            options.push({ text: 'default', value: 'default', selected: false });
+          } else if (regex && regex.exec('default')) {
+            options.push({ text: 'default', value: 'default', selected: false });
+          }
+        }
```

Comment:
I would like some test coverage for:
- When a variable has `default` in regex and data source is default
- When a variable hasn't have `default` in regex and data source is default

Reply:
done